### PR TITLE
EMBPD00161219

### PR DESCRIPTION
**Pull request by LNSimha**
[SAP-ITS][IE][CE5] - SIP is not shown while trying to enter text in textbox of sap-its pages

### CODE REVIEW

**Comment by LNSimha** [comment]
@darryncampbell
@joydeep12 
@abhineetagarwal 

Review this

File: platform/wm/rhodes/browser/CEBrowserEngine.cpp [hunk 1]
```diff
@@ -87,6 +87,40 @@ LRESULT CEBrowserEngine::CreateEngine()
...
 
+
+	TCHAR szPlatform[128];
+	memset(szPlatform, 0, 127 * sizeof(TCHAR));
+	SystemParametersInfo(SPI_GETOEMINFO, 127, szPlatform, 0);
+	TCHAR szSIPValue[100];
+	ZeroMemory(szSIPValue,100);
+
+
+		if((_tcsstr(szPlatform,TEXT("MK4000")) != NULL) || (_tcsstr(szPlatform,TEXT("BigBoardMPA3")) != NULL) || (_tcsstr(szPlatform,TEXT("MC18")) != NULL))
```
Comment (abhineetagarwal) [inline platform/wm/rhodes/browser/CEBrowserEngine.cpp]: Will it not be added for MK3000 device?

Comment (LNSimha) [inline platform/wm/rhodes/browser/CEBrowserEngine.cpp]: I don't have the device. If anybody have the OEM name then give it to me I can add it.

Comment (darryncampbell) [inline platform/wm/rhodes/browser/CEBrowserEngine.cpp]: Just use MK3100 and MK3000... not sure what 'BigBoardMPA3' is but I suspect it is a pre-release BSP

Comment (LNSimha) [inline platform/wm/rhodes/browser/CEBrowserEngine.cpp]: Updated with MK3100 and MK3000

**Comment by LNSimha** [comment]
@darryncampbell  replace string for MK3100 and added MK3000 in condition

**Comment by darryncampbell** [comment]
Thanks.  Please remove references to szSIPValue, I presume that is just for debugging.

**Comment by LNSimha** [comment]
@darryncampbell  Removed references

**Comment by darryncampbell** [comment]
Looks good, thanks.

**Pull request merged**